packages/server-core/src/media/static-resource/static-resource.hooks.ts

```diff
@@ -80,24 +103,64 @@ export default {
       schemaHooks.resolveQuery(staticResourceQueryResolver)
     ],
     find: [
-      iff(isProvider('external'), verifyScope('static_resource', 'read')),
+      iff(
+        isProvider('external'),
+        iffElse(
+          checkScope('static_resource', 'read'),
+          [],
+          [verifyScope('editor', 'write'), resolveProjectId(), verifyProjectPermission(['owner', 'editor', 'reviewer'])]
+        )
+      ),
       discardQuery('action'),
+      discardQuery('project'),
+      discardQuery('projectId'),
```

Comment:
These 3 can be done in single line as:
```suggestion
      discardQuery('action', 'project', 'projectId')
```

Reply:
Done!